module/modmanager/manager.go

```diff
@@ -348,7 +348,7 @@ func (mgr *Manager) startModule(ctx context.Context, mod *module) error {
 
 	// create the module's data directory
 	if mod.dataDir != "" {
-		mgr.logger.Infof("Creating data directory %q for module %q", mod.dataDir, mod.cfg.Name)
+		mgr.logger.Debugf("Creating data directory %q for module %q", mod.dataDir, mod.cfg.Name)
```

Comment:
Happens too often and is not interesting enough to be `Info` level IMO.

Reply:
Agreed. I think we can afford one log line per module addition. This isn't what I'd spend that budget on.